src/components/track/helpers.ts

```diff
@@ -45,6 +45,7 @@ export const getCollectionWithFallback = (collection: Collection | null) => {
       is_delete: false,
       activity_timestamp: '',
       _co_sign: undefined,
+      playlist_owner_id: -1,
```

Comment:
could this break anything? maybe some selector assuming if (playlist_owner_id) or something

Reply:
I don't think so - this should basically just return a dummy collection, but we forgot to include owner_id bc we didn't use it anywhere on the dummy collection